src/main/scripts/src/app/workspace/workspace-routing.module.ts

```diff
@@ -21,15 +21,64 @@
 import {NgModule} from '@angular/core';
 import {RouterModule, Routes} from '@angular/router';
 import {WorkspaceChooserComponent} from './workspace-chooser/workspace-chooser.component';
-import {OrganizationEditComponent} from './organization-edit/organization-edit.component';
-import {ProjectEditComponent} from './project-edit/project-edit.component';
+import {OrganizationFormComponent} from './organization/form/organization-form.component';
+import {ProjectFormComponent} from './project/form/project-form.component';
+import {OrganizationSettingsComponent} from './organization/organization-settings.component';
+import {ProjectSettingsComponent} from './project/project-settings.component';
+import {ProjectPermissionsComponent} from './project/permissions/project-permissions.component';
+import {OrganizationPermissionsComponent} from './organization/permissions/organization-permissions.component';
 
 const workspaceRoutes: Routes = [
-  {path: 'workspace', component: WorkspaceChooserComponent},
-  {path: 'organization/:organizationCode/project/:projectCode/edit', component: ProjectEditComponent},
-  {path: 'organization/:organizationCode/project/add', component: ProjectEditComponent},
-  {path: 'organization/:organizationCode/edit', component: OrganizationEditComponent},
-  {path: 'organization/add', component: OrganizationEditComponent}
+  {
+    path: 'organization/:organizationCode/project/:projectCode',
+    component: ProjectSettingsComponent,
+    children: [
+      {
+        path: 'settings',
+        component: ProjectFormComponent
+      },
+      {
+        path: 'permissions',
+        component: ProjectPermissionsComponent
+      },
+      {
+        path: '',
+        redirectTo: 'settings',
+        pathMatch: 'full'
+      }
+    ]
+  },
+  {
+    path: 'organization/:organizationCode/project/add',
```

Comment:
You need to put this path before the previous one, otherwise it does not work because `add` is used as `projectCode` parameter.